bigbluebutton-html5/imports/startup/server/logger.js

```diff
@@ -18,46 +18,17 @@ Logger.configure({
 
 Meteor.startup(() => {
   const LOG_CONFIG = Meteor.settings.private.log || {};
-  let { filename } = LOG_CONFIG;
```

Comment:
I think you also have to remove this from the `settings-development.json` and `settings-production.json`

Reply:
@OZhurbenko Done.